CHANGELOG.md

```diff
@@ -1,5 +1,129 @@
 # Node.js ChangeLog
 
+## 2016-04-20, Version 5.11.0 (Stable), @thealphanerd
+
+## Notable Changes
+
+* **Buffer**:
+  * `Buffer.prototype.compare` can now compare sub-ranges of two Buffers. (James M Snell) [#5880](https://github.com/nodejs/node/pull/5880)
+* **deps**:
+  * update to http-parser 2.7.0 (Fedor Indutny) [#6279](https://github.com/nodejs/node/pull/6279)
+  * update ESLint to 2.7.0 (silverwind) [#6132](https://github.com/nodejs/node/pull/6132)
+* **net**:
+  * adds support for passing DNS lookup hints to `createConnection()` (Colin Ihrig) [#6000](https://github.com/nodejs/node/pull/6000) 
+* **node**:
+  * Make the builtin libraries available for the `--eval` and `--print` CLI options (Anna Henningsen) [#6207](https://github.com/nodejs/node/pull/6207)
+* **npm**:
+  * upgrade npm to 3.8.6 (Kat Marchán) [#6153](https://github.com/nodejs/node/pull/6153)
+* **repl**:
+  * Pressing enter in the repl will repeat the last command by default if no input has been received. This behaviour was in node previously and was not removed intentionally. (Rich Trott) [#6090](https://github.com/nodejs/node/pull/6090)
```

Comment:
Nit, and I really mean nit, feel free to ignore, but if you like addressing the tiniest details: `behaviour` -> `behavior`. I believe the Docs WG standardized on US English spellings, but even if they haven't, it certainly is the _de facto_ standard. For example, `behavior` appears five times in the current DNS API doc, and `behaviour` does not appear at all.